pages/index.js

```diff
@@ -1,28 +1,79 @@
 import React from 'react'
 import glamorous from 'glamorous'
 import Logo from '../components/glamorous-logo'
-import {Button, SecondaryButton} from '../components/styled-links'
+import {Button} from '../components/styled-links'
 import Layout from '../components/layout'
+import Separator from '../components/separator'
+import CodePreview from '../components/code-preview'
 
 const {Div} = glamorous
 
-export default Root
+const Title = glamorous.h1((props, {colors}) => ({
+  margin: '1em 0',
+  color: colors.primaryMed,
+  fontWeight: 'normal',
+  fontSize: '1em'
+}))
 
-function Root() {
+const CodePreviewWrapper = glamorous.div((props, {colors}) => ({
+  position: 'relative',
+  padding: '1em',
+  background: 'transparent',
+  '::after': {
+    content: '""',
+    top: '8em',
+    left: 0,
+    bottom: 0,
+    right: 0,
+    position: 'absolute',
+    background: colors.primaryMed,
+    zIndex: -1
+  }
+}))
+
+const UsersHeading = glamorous.div({
+  textTransform: 'uppercase',
+  color: '#fff',
+  fontSize: '0.8rem',
+  fontWeight: '600',
+  margin: '2.5rem 0 0.5rem',
+  opacity: 0.8,
+  textAlign: 'center'
+})
+
+const Container = glamorous.div({
+  height: 500,
+  background: 'white'
+})
+
+export default () => {
   return (
     <Layout>
-      <Div margin="20px auto" maxWidth={700} textAlign="center">
-        <Logo marginTop={50} />
-        <Div marginTop={30}>
-          Maintainable CSS with React
+      <Div margin={20}>
```

Comment:
Can we change the inline syntax to use the `glamorous` factory? I'd also like to keep an eye on `margin` and `padding` to ensure a consistent style.